### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by v-tashish**

### CODE REVIEW

File: base/gpu/openmpi4.1.0-cuda11.1-cudnn8-ubuntu20.04/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 -  **Image Name**: docker pull mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.1-cudnn8-ubuntu20.04 :
 
+
+
```
Comment (Kmumuda) [inline base/gpu/openmpi4.1.0-cuda11.1-cudnn8-ubuntu20.04/release-notes.md:4]: remove these extra spaces two lines

File: base/gpu/openmpi4.1.0-cuda11.0.3-cudnn8-ubuntu18.04/release-notes.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
 
+:20220708.v1
+------------------- 
+-   **Dependencies** :
+
+-  [CVE-2019-20838](https://ubuntu.com/security/notices/USN-5425-1)
+-  [CVE-2022-1664](https://ubuntu.com/security/notices/USN-5446-1)
+-  [CVE-2019-13050](https://ubuntu.com/security/notices/USN-5431-1)
+-  [CVE-2022-1304](https://ubuntu.com/security/notices/USN-5464-1)
+-  [CVE-2022-2068 ](https://ubuntu.com/security/notices/USN-5488-1)
```
Comment (Kmumuda) [inline base/gpu/openmpi4.1.0-cuda11.0.3-cudnn8-ubuntu18.04/release-notes.md:11]: remove this extra space here

File: base/gpu/openmpi4.1.0-cuda11.1-cudnn8-ubuntu20.04/release-notes.md (hunk 1 above)
Comment (Kmumuda) [reply base/gpu/openmpi4.1.0-cuda11.1-cudnn8-ubuntu20.04/release-notes.md:4]: remove this one extra line here

File: base/cpu/openmpi3.1.2-ubuntu18.04/release-notes.md [hunk 3]
```diff
@@ -1,5 +1,14 @@
 -  **Image Name**: docker pull mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04 :
 
+:20220708.v1
+------------------- 
+-   **Dependencies** :
+-   [CVE-2022-1304 ](https://ubuntu.com/security/notices/USN-5464-1)
```
Comment (Kmumuda) [inline base/cpu/openmpi3.1.2-ubuntu18.04/release-notes.md:6]: remove this extra space

**Review by satudum — APPROVED** [review]

**Review by VenkateshG25 — APPROVED** [review]

**Review by Kmumuda — APPROVED** [review]

File: base/cpu/openmpi3.1.2-ubuntu18.04/release-notes.md (hunk 3 above)
Comment (v-tashish) [reply base/cpu/openmpi3.1.2-ubuntu18.04/release-notes.md]: Done with the changes.

File: base/cpu/openmpi4.1.0-ubuntu20.04/release-notes.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
 -  **Image Name**: docker pull mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04 :
 
+:20220708.v1
+------------------- 
+-   **Dependencies** :
+-   [CVE-2019-20838](https://ubuntu.com/security/notices/USN-5425-1) 
+-   [CVE-2022-1664](https://ubuntu.com/security/noties/USN-5446-1) 
+-   [CVE-2022-1304](https://ubuntu.com/security/notices/USN-5464-1) 
+-   [CVE-2022-2068)](https://ubuntu.com/security/notices/USN-5488-1)
```
[the remark this answers is not in the document]
Comment (v-tashish) [reply base/cpu/openmpi4.1.0-ubuntu20.04/release-notes.md]: Done with the changes.

**Pull request closed without merging**